app/components/CV/experience.tsx

```diff
@@ -0,0 +1,27 @@
+import { ExperienceType } from './types'
+
+const Experience = ({ experience }: { experience: ExperienceType[] }) => {
+  return (
+    <div className="my-8">
+      <h3 className="text-sm uppercase font-bold py-2">experience</h3>
+      {experience.map((work, index) => {
+        return (
```

Comment:
**nitpick**: Consider using a more descriptive key for the map function, such as `work.id`, if available, instead of the index to avoid potential issues with reordering.